Source/FieldSolver/SpectralSolver/SpectralKSpace.cpp

```diff
@@ -205,6 +205,13 @@ SpectralKSpace::getModifiedKComponent( const DistributionMapping& dm,
                 }
             }
         }
+        if (nodal){
+            if (i_dim == 0){
+                modified_k[k.size()-1] = 0.0_rt;
+          }   else {
+                modified_k[k.size()/2] = 0.0_rt;
+          }
+        }
```

Comment:
@oshapoval @RemiLehe Could you explain why `i_dim == 0` (x) is treated in one way, and `i_dim != 0` (y,z) differently here?

Reply:
Yes, I'll had some comments to the PR.

Reply:
Thanks Remi for the comments!